Dockerfile

```diff
@@ -20,9 +20,6 @@ RUN echo "gem: --no-ri --no-rdoc" > ~/.gemrc && \
 # Install git. (Jekyll expects it.)
 RUN apk add --no-cache git
 
-# Set environment variables that GitHub Pages expects.
-ENV PAGES_REPO_NWO googleapis/googleapis.github.io
-
```

Comment:
Removing this from the docker file so that it can be more easily configured when initializing. See changes to `serve.sh`.